wcfsetup/install/files/acp/update_com.woltlab.wcf_5.3_style.php

```diff
@@ -32,7 +31,7 @@
 	
 	// 3) If the file had a custom image folder we need to copy all files into the asset folder.
 	// Moving the files is unsafe, because multiple styles can share a single image folder.
-	if ($style->imagePath != 'images/') {
+	if (!empty($style->imagePath) && $style->imagePath != 'images/') {
 		$srcPath = FileUtil::addTrailingSlash(WCF_DIR.$style->imagePath);
```

Comment:
```suggestion
	$srcPath = FileUtil::addTrailingSlash(WCF_DIR.$style->imagePath);
	if ($srcPath !== WCF_DIR && $srcPath !== WCF_DIR.'images/') {
```

I hope this suggestion applies well. My suggestion also avoids issues with differences in trailing slashes.

Reply:
Good idea. I changed the commit like that.